go.mod

```diff
@@ -9,6 +9,10 @@ require (
 	github.com/hashicorp/hcl/v2 v2.3.0 // indirect
 	github.com/hashicorp/terraform-config-inspect v0.0.0-20191212124732-c6ae6269b9d7 // indirect
 	github.com/hashicorp/terraform-plugin-sdk v1.8.0
+	github.com/kr/pretty v0.2.0
 	github.com/vmware/go-vcloud-director/v2 v2.8.0-beta.2
 	golang.org/x/crypto v0.0.0-20200510223506-06a226fb4e37 // indirect
 )
+
+// replace github.com/vmware/go-vcloud-director/v2 => ../go-vcloud-director
+replace github.com/vmware/go-vcloud-director/v2 => github.com/dataclouder/go-vcloud-director/v2 v2.5.0-alpha.4.0.20200816180754-4de23ed92595
```

Comment:
I assume this will be removed before merge to master?
Is there a pre-version of go-vcloud-director that support acl and the new hot plug parameter ?

Reply:
go.mod is changed to using the latest go-vcloud-director tag once the depending PR ([PR #329](https://github.com/vmware/go-vcloud-director/pull/329)) is merged to master.
The only ACL that I know about is the one that supports this PR.

Reply:
Nevermind. I rebased this PR and https://github.com/vmware/go-vcloud-director/pull/329 on master, build the go-vcloud-director and I know have a vcd provider to test hot plug and this PR in one binary :-)